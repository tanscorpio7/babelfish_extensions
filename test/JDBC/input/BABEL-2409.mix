-- tsql
USE master
go

-- tsql
CREATE SCHEMA abc;
GO

-- psql
DROP SCHEMA master_abc;
GO

-- psql
ALTER SCHEMA master_abc RENAME TO abc;
GO

-- psql
CREATE SCHEMA master_xyz;
GO
ALTER SCHEMA master_xyz OWNER TO master_dbo;
GO

-- tsql
DROP SCHEMA xyz;
GO

-- psql
DROP SCHEMA master_xyz;
GO

-- tsql
DROP SCHEMA abc;
GO
