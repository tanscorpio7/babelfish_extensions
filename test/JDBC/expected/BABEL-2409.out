-- tsql
USE master
go

-- tsql
CREATE SCHEMA abc;
GO

-- psql
DROP SCHEMA master_abc;
GO
~~ERROR (Code: 0)~~

~~ERROR (Message: ERROR: Could not drop schema created under T-SQL dialect: "master_abc"
    Server SQLState: XX000)~~


-- psql
ALTER SCHEMA master_abc RENAME TO abc;
GO
~~ERROR (Code: 0)~~

~~ERROR (Message: ERROR: Could not rename schema created under T-SQL dialect: "master_abc"
    Server SQLState: XX000)~~


-- psql
CREATE SCHEMA master_xyz;
GO
ALTER SCHEMA master_xyz OWNER TO master_dbo;
GO

-- tsql
DROP SCHEMA xyz;
GO
~~ERROR (Code: 33557097)~~

~~ERROR (Message: Could not drop schema created under PostgreSQL dialect: "master_xyz")~~


-- psql
DROP SCHEMA master_xyz;
GO

-- tsql
DROP SCHEMA abc;
GO
